cli/flags.rs

```diff
@@ -764,6 +766,12 @@ Ignore formatting a file by adding an ignore comment at the top of the file:
         .help("Check if the source files are formatted")
         .takes_value(false),
     )
+    .arg(
+      Arg::with_name("stdin-markdown")
+        .long("stdin-markdown")
+        .help("Consider standard input (stdin) as markdown")
```

Comment:
Does it make more sense to be --stdin-ext=md ? Or something similar... rather than a bool?

Reply:
It does, but I'm a little concerned that the flag might confuse users into thinking that it can accept other extensions too. Right now, TS(X)/JS(X) are autodetected and require no input from users for formatting to work properly. I thought to go with the same route but some inputs from @bartlomieju, and the `--ts` flag of `deno eval` motivated me to go with this simple solution. 

I feel like `--stdin-markdown` is more explicit and leads to less confusion. Let me know your thoughts. :)

Reply:
Maybe `--ext={js|jsx|ts|tsx|md}`? (though I'm not sure if there's any difference for js(x) and ts(x) files. @lucacasonato wdyt?

Reply:
We should use a global solution that could potentially be used by `deno run -`, `deno lint -`, `deno eval` to choose between js and ts. I don't think `deno eval --ts` was good. I think we want some abbreviation of `--anonymous-media-type`.

Reply:
I'm +1 for a global solution. What do you'll think of `--content-type={js|jsx|ts|tsx|md}`?

> (though I'm not sure if there's any difference for js(x) and ts(x) files.

There's no difference as they're all processed using `dprint-plugin-typescript`.

Reply:
> I think we want some abbreviation of `--anonymous-media-type`.

I'm not sure I understand this bit Nayeem, care to explain? 

> I'm +1 for a global solution. What do you'll think of --content-type={js|jsx|ts|tsx|md}?

`--content-type` sounds good, but I still like `--ext` better. 

We need more opinions on this flag name.

Reply:
Since the flag will be used across commands, your (@lucacasonato, @ry, @kitsonk, @kt3k) thoughts/feedback will be helpful.

Reply:
> We should use a global solution that could potentially be used by `deno run -`, `deno lint -`, `deno eval` to choose between js and ts. I don't think `deno eval --ts` was good. I think we want some abbreviation of `--anonymous-media-type`.

Discussed offline with @lucacasonato and we think that `--ext` is preferable. @nayeemrmn I do agree that `deno eval --ts` was a mistake. I'm still not sure what you meant by `--anonymous-media-type`...

Reply:
`--ext` is fine, my point was just to say that the flag should work to provide the content type of any anonymous module (like eval, stdin) across cli.

Reply:
Let's address that in follow up PRs